### DISCUSSION TITLE
`export mavenrepo` and GAV settings

### DISCUSSION THREAD

**Discussion by quintesse** [jbangdev/jbang#1127] [General]
Why does `export mavenrepo` use system properties to define groupId, artifactId and version? Why doesn't it simply use command line flags? That way we have the documentation and we can also enforce that the user provides that information, because the defaults (`g:a:999-SNAPSHOT`) won't ever be usable.

**Comment by maxandersen** [comment]
Just because it was minimal that could work and the system property could be use as fallbacks in command line once we got further. 

I did also want to add a //GAV g:a:v Source command. Preferably one Where v if blank gets to be 999-snapshot or some other snapshotty content.

**Reply by quintesse** [reply]
Ok, so now with the [Configuration PR](#604) we can simply add command line options and get the System properties "for free".
